foundry.toml

```diff
@@ -4,7 +4,7 @@ out = 'out'
 libs = ['lib']
 via_ir = true
 solc_version = '0.8.17'
-optimizer_runs = 1_000_000
+optimizer_runs = 10_000
```

Comment:
were you getting better results with this?

Reply:
its because bytecode size went up so much we could no longer have 1M runs 😢 will look into getting bytecode down next week so we can put this back up